sa/rate_limits_test.go

```diff
@@ -88,33 +86,29 @@ func TestCertsPerNameRateLimitTable(t *testing.T) {
 }
 
 func TestNewOrdersRateLimitTable(t *testing.T) {
-	if !strings.HasSuffix(os.Getenv("BOULDER_CONFIG_DIR"), "config-next") {
-		return
-	}
-
 	sa, _, cleanUp := initSA(t)
 	defer cleanUp()
 
 	zeroCountRegID := int64(1)
 	manyCountRegID := int64(2)
 
-	start := time.Now().Truncate(time.Hour)
+	start := time.Now().Truncate(time.Minute)
 	for i := 0; i <= 10; i++ {
 		tx, err := sa.dbMap.Begin()
 		test.AssertNotError(t, err, "failed to open tx")
-		err = addNewOrdersRateLimit(context.Background(), tx, manyCountRegID, start.Add(time.Hour*time.Duration(i)))
+		err = addNewOrdersRateLimit(context.Background(), tx, manyCountRegID, start.Add(time.Minute*time.Duration(i)))
```

Comment:
Note that this test does not try to insert two events in the same minute, thus not covering the `ON DUPLICATE KEY` "codepath" of the `addNewOrdersRateLimit` method.